packages/atomic/src/components/search/atomic-search-interface/analytics-config.ts

```diff
@@ -2,7 +2,7 @@ import {
   AnalyticsConfiguration,
   SearchEngineConfiguration,
 } from '@coveo/headless';
-import {AnalyticsClientSendEventHook} from '@coveo/headless/node_modules/coveo.analytics';
+import {AnalyticsClientSendEventHook} from '../../../../../../node_modules/coveo.analytics';
```

Comment:
just `import {AnalyticsClientSendEventHook} from 'coveo.analytics';` should work ? 🤔

Reply:
"coveo.analytics" is extraneous, it's not a direct dependency, but through headless

Reply:
> "coveo.analytics" is extraneous,

I think either headless should export AnalyticsClientSendEventHook, or you should install coveo.analytics in this workspace.

Reply:
Yeah should be a direct dependency.

We can `import type` to be explicit that it's just for types, not code.

OR : Re-export the type from headless (since it is a dependency from Headless)
That second option would make sense, since it's actually a type that you need to know about if you implement on top of Headless (as showcased here)